.gitlab-ci.yml

```diff
@@ -3,30 +3,30 @@ stages:
   - build
 
 clippy:
-  image: paritytech/ci-linux:c4f0daf3-20220110
+  image: paritytech/ci-linux:f203f298-20220421
```

Comment:
note: this doesn't change the rust version inside the CI. So it was a dependency issue.

Reply:
Yeah seems like. I'm just happy it works now.